src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -8,12 +8,19 @@
   public WasapiDownloaderSettings settings = null;
 
 
-  public WasapiDownloader(String settingsFileLocation, String[] args) throws IOException {
+  public WasapiDownloader(String settingsFileLocation, String[] args) throws SettingsLoadException {
     settings = new WasapiDownloaderSettings(settingsFileLocation, args);
   }
 
+  public void executeFromCmdLine() {
+    if (settings.shouldDisplayHelp()) {
+      System.out.print(settings.getHelpAndSettingsMessage());
+      return;
+    }
```

Comment:
just to be clear:  this method (perhaps needs a TODO) will include code to do stuff based on the command line arguments, yes?